README.md

```diff
@@ -131,23 +115,23 @@ This gets really messy with callbacks. But with `Deferred`, it's just:
 
 ```swift
 // Let's use type annotations to make it easier to see what's going on here.
-let friends: Deferred<[Friend]> = AsynchronousFriends(forUser: jimbob)
-let names: Deferred<[Name]> = friends.flatMap { friends in
+let friends: Deferred<[Friend]> = fetchFriends(forUser: jimbob)
+let names: Future<[Name]> = friends.flatMap { (friends: [Friend]) -> Future<[Name]> in
     // fork: get an array of not-yet-determined names
     let names: [Deferred<Name>] = friends.map { AsynchronousCall(.Name, friend: $0) }
 
     // join: get a not-yet-determined array of now-determined names
-    let allNames: Deferred<[Name]> = all(names)
-    return allNames
+    return names.joinedValues
 }
-names.upon { names: [Name] in
+
+names.upon { (names: [Name]) in
     // names has been determined - use it!
     dataSource.array = names
     tableView.reloadData()
 }
 ```
 
-## Tasks
+## Basic Tasks
 
 ### Vending a Future
```

Comment:
Now that we have an actual `Future` type, we should probably be more careful about Future/Deferred verbiage. We probably do want to counsel them to vend a Future and not a Deferred, though.